data/mods/Xedra_Evolved/mutations/mutations.json

```diff
@@ -63,6 +63,33 @@
     "valid": false,
     "description": "You feel like you've left behind your old life and form."
   },
+  {
+    "type": "mutation",
+    "id": "DRUNKEN",
+    "name": { "str": "Drunken Master" },
+    "points": 1,
+    "description": "The ancient arts of drunken brawling come naturally to you!  While under the influence of alcohol, your melee skill will rise considerably, especially unarmed combat.",
+    "starting_trait": true,
+    "valid": false,
+    "cancels": [ "LIGHTWEIGHT" ]
+  },
+  {
+    "type": "mutation",
+    "id": "KI_STRIKE",
+    "name": { "str": "Ki Strike" },
+    "points": 2,
+    "description": "Who needs weapons?  You deal more melee damage while fighting barehanded and without gloves.  This damage improves as your unarmed skill increases.",
+    "starting_trait": true,
+    "valid": false
+  },
+  {
+    "type": "mutation",
+    "id": "EXTEND_LIFE",
+    "name": { "str": "Extended Life" },
+    "points": 0,
+    "description": "You have imbibed a potion that proclaims to extend your life.  You feel healthier at least.",
```

Comment:
```suggestion
    "description": "You swallow the potion and feel younger already.  Though the taste lingers long past when you'd hope it would pass.",
```